src/drivers/tty/serial/idesc_serial.h

```diff
@@ -14,4 +14,6 @@ struct uart;
 
 extern struct idesc *idesc_serial_create(struct uart *uart, mode_t mod);
 
+extern const struct idesc_ops *idesc_serial_get_ops();
```

Comment:
If argument list is empty, function can get any number of arguments, so it's better to write `idesc_serial_get_opts(void)` explicitly.